coralproject/talk
# [CORL-1292] Set better aria labels on the comment action buttons

## What does this PR do?

Adds more descriptive aria labels to the comment action buttons like React, Share, and Reply.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to comment stream
- Right click on an action button within a comment
- Inspect element
- See that the button element has an aria label similar to `Reply to comment by <username>`
